src/dotnet/commands/dotnet-install/LocalizableStrings.resx

```diff
@@ -172,4 +172,10 @@ The error was:
   <data name="InstallToolVersionDefinition" xml:space="preserve">
     <value>Version of the package in NuGet</value>
   </data>
+  <data name="GlobalOptionDescription" xml:space="preserve">
+    <value>Install user wide.</value>
```

Comment:
Unless I was a user coming from node.js, I'm going to find this option perplexing.  Installing "globally" means to install "user wide"?  Given that we have no "local" install to default to, a required boolean option strikes me as bad UX.  I get that we're trying to future-proof a default of local install (is that something that even makes sense for tools? it does for npm which also manages runtime package dependencies), but this seems premature to me.

Reply:
@KathleenDollard on -g

Reply:
@peterhuene 

Yes, --user would be more correct than --global, but...

It's not coming from node.js, it's coming from npm experience, which happens to be deployed by node. I make that distinction because everybody (even us - Azure CLI as example) has the install node to run npm experience. So paralleling that is important. 

We plan to have a "session" based tool that will be the default. I anticipate that in this release, but I don't think Preview 1. That will install into the user's session. 

"Local" doesn't make sense to me. There is my whole experience (global), this session (planned) and this repo (in future thinking, not designed, not this version). 

If global was default, we could not change later. 

So, yes, this is odd, but I think correct.

Reply:
I totally get what you're saying, but I don't think I share the sentiment that we need to be consistent with a package manager from another ecosystem, even if we've been instructing users to use it for our own tools prior to this feature.  If it truly makes sense for `dotnet` for the global option to be non-default, then I'm all for this.